common/source/docs/common-ekf-sources.rst

```diff
@@ -27,4 +27,10 @@ Three set of EKF position and velocity source parameters are provided which can
 
 This feature is especially helpful when using :ref:`common-non-gps-to-gps`.
 
+Source Switching
```

Comment:
I think this title should be updated to EK3_SRC_OPTIONS? .. or something like that?  otherwise we have two identical titles.

Reply:
I fixed this and some other things